### DISCUSSION TITLE
FR: Locked Move to One Dimension

### DISCUSSION THREAD

**Discussion by Remodor** [PintaProject/Pinta#1183] [Ideas]
In many apps, like Krita, LibreOffice, Paint 3D (not Paint though 😄), draw.io and probably a bunch I cannot remember, it is possible to use shift to lock a move to one dimension. This is very helpful for precisely moving a selection to a location and keeping a relation to the canvas or other parts of the image.

It would be nice to have this feature in Pinta too.

**Comment by cameronwhite** [comment]
This may also be related to #585 where for scaling the proposed idea is to add handles at the corners for resizing. For moving along an axis, there could also be arrow handles to drag in a specific direction as an alternative option to using a keyboard shortcut
